featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -569,12 +576,15 @@ def parallel_calculate_chunks(chunks, features, approximate, training_window,
         pickled_feats = cloudpickle.dumps(features)
         _saved_features = client.scatter(pickled_feats)
         client.replicate([_es, _saved_features])
+        num_scattered_workers = len(client.who_has([Future(es_token)]).get(es_token, []))
+        num_workers = len(client.scheduler_info()['workers'].values())
+
+        scatter_warning(num_scattered_workers, num_workers)
         if verbose:
             end = time.time()
             scatter_time = end - start
-            scatter_string = "EntitySet scattered to workers in {:.3f} seconds"
-            print(scatter_string.format(scatter_time))
-
+            scatter_string = "EntitySet scattered to {} workers in {:.3f} seconds"
```

Comment:
The issue wanted to remove the decimal part of the "in X seconds" message.  Maybe round to integer and if it took less than 1 second  display "<1" or "under 1"?

Reply:
I added in the rounding but I decided to not add in the "<1" so that I don't have to deal with a complex code cov situation. In theory I could create a context manager to time it, and then run an on complete function which then prints out a special string but it feels too complex for a very minor piece of functionality.

Reply:
I think it'd be easiest to just round up to the near integer seconds. This is just diagnostic info, so +/- 1 second doesn't matter that much